src/main.js

```diff
@@ -17,6 +17,10 @@ var imageInput = document.querySelector("#poster-image-url");
 var titleInput = document.querySelector("#poster-title");
 var quoteInput = document.querySelector("#poster-quote");
 var showPoster = document.querySelector(".make-poster");
+
+var savePoster = document.querySelector(".save-poster");
+var posterGrid = document.querySelector(".saved-posters-grid");
+var clonedPoster = document.querySelector(".poster");
```

Comment:
Like your naming conventions, made it very easy for me to identify relationships.